src/Integration.TeamExplorer/CommonStyles.xaml

```diff
@@ -1,58 +1,45 @@
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:po="http://schemas.microsoft.com/winfx/2006/xaml/presentation/options"
+                    xmlns:tfconverters="clr-namespace:Microsoft.TeamFoundation.Controls.WPF.Converters;assembly=Microsoft.TeamFoundation.Controls"
+                    xmlns:vsutilities="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Utilities">
 
     <ResourceDictionary.MergedDictionaries>
-        <ResourceDictionary Source="pack://application:,,,/Microsoft.TeamFoundation.Controls;component/WPF/SharedResources.xaml"/>
+        <ResourceDictionary
+            Source="pack://application:,,,/Microsoft.TeamFoundation.Controls;component/WPF/SharedResources.xaml" />
     </ResourceDictionary.MergedDictionaries>
 
     <!-- Converters -->
-    <tfconverters:AnyTrueToVisibleConverter x:Key="AnyTrueToVisibleConverter"/>
-    <tfconverters:TrueToVisibleConverter x:Key="TrueToVisibleConverter"/>
-    <tfconverters:IntToBoolConverter x:Key="IntToBoolConverter"/>
-    <tfconverters:IsNullConverter x:Key="IsNullConverter"/>
-    <vsutilities:BrushToColorConverter x:Key="BrushToColorConverter"/>
+    <tfconverters:AnyTrueToVisibleConverter x:Key="AnyTrueToVisibleConverter" />
+    <tfconverters:TrueToVisibleConverter x:Key="TrueToVisibleConverter" />
+    <tfconverters:IntToBoolConverter x:Key="IntToBoolConverter" />
+    <tfconverters:IsNullConverter x:Key="IsNullConverter" />
+    <vsutilities:BrushToColorConverter x:Key="BrushToColorConverter" />
 
     <!-- Team Explorer Controls -->
     <Style x:Key="SQTESectionStyle" TargetType="FrameworkElement">
-        <Setter Property="Margin" Value="6,0"/>
+        <Setter Property="Margin" Value="6,0" />
     </Style>
 
     <!-- Icon brushes -->
-    <SolidColorBrush x:Key="SQForegroundBrush" Color="#ff579dd4" po:Freeze="True"/>
-    <SolidColorBrush x:Key="SQBackgroundBrush" Color="White" po:Freeze="True"/>
-
-    <!-- Icon geometry -->
-    <Geometry x:Key="SQServerIconGeometry" po:Freeze="True">
-        F1
-        M 12.34,12.68 C 12.34,7.07 7.71,2.5 2,2.5
-        M 13.42,9 C 12.59,5.49 9.75,2.56 6.18,1.55
-        M 14.11,5.77 C 13.26,3.91 11.8,2.29 10.02,1.2
-    </Geometry>
-
-    <!-- Icon pens -->
-    <Pen x:Key="SQIconGeomPen"
-         Brush="{StaticResource SQForegroundBrush}"
-         Thickness="1"/>
-    <Pen x:Key="SQIconGeomOutlinePen"
-         Brush="{StaticResource SQBackgroundBrush}"
-         Thickness="3"/>
-    <Pen x:Key="SQIconGeomOutlineRoundedPen"
-         Brush="{StaticResource SQBackgroundBrush}"
-         Thickness="3"
-         StartLineCap="Round"
-         EndLineCap="Round"/>
-
+    <SolidColorBrush x:Key="SQForegroundBrush" Color="#ff579dd4" po:Freeze="True" />
+    <SolidColorBrush x:Key="SQBackgroundBrush" Color="White" po:Freeze="True" />
     <!-- Icon drawing images -->
     <DrawingImage x:Key="SonarQubeServerIcon" po:Freeze="True">
```

Comment:
While the SonarQube Cloud looks great. It seems that the SonarQube Server icon is not centered with the radio button and it looks too close to the text above.

![image](https://github.com/user-attachments/assets/b4449c4c-d197-4cf4-96dc-8756b98e139f)